src/interfaces/python/swig_typemaps.i

```diff
@@ -1315,7 +1324,15 @@ def _internal_factory_wrapper(object_name, new_name, docstring=None):
 
     return _internal_factory
 
+distance = _internal_factory_wrapper("_distance", "distance")
```

Comment:
do you think it would be possible to have a single list of those factory methods that is then used? This way we have everything twice? Or is that not nice to read?

Reply:
Do you mean put all the names of the factory methods in a list and then pass it to a loop?

Reply:
Yes rxactly

Reply:
Yup, that's doable. I can push some code that does that. Basically _swig_monkey_patch checks if the first argument is a module, and then adds the object with the appropriate Python C-API.